src/ariel-os-embassy/src/delegate.rs

```diff
@@ -41,9 +43,10 @@ pub struct Delegate<T> {
     send: Signal<CriticalSectionRawMutex, SameExecutorCell<*mut T>>,
     reply: Signal<CriticalSectionRawMutex, ()>,
     was_exercised: AtomicBool,
+    _not_send: PhantomData<T>,
```

Comment:
Is it desirable that this type will now be `Send` if `T` is `Send`? It wasn't before.

Reply:
no, my bad. thanks for noticing!